.github/workflows/e3sm-gh-pages.yml

```diff
@@ -12,28 +12,46 @@ on:
 concurrency:
   # Prevent 2+ copies of this workflow from running concurrently
   group: e3sm-docs-action
-  
+
 jobs:
-  Build-and-Deploy-docs:
+  e3sm-docs:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v4
+      - 
+        name: Check out code
+        uses: actions/checkout@v4
         with:
           show-progress: false
-          fetch-depth: 0 # Needed, or else gh-pages won't be fetched, and push rejected
+          # We will need submodules when/if we build EAMxx docs here
           submodules: false # speeds up clone and not building anything in submodules
-      - name: Show action trigger
+      - 
+        name: Show action trigger
         run: echo "= The job was automatically triggered by a ${{github.event_name}} event."
```

Comment:
Nitpick: I would prune "automatically", since the workflow can be manually triggered.